packages/ListBox/src/components/Trigger/Trigger.js

```diff
@@ -88,7 +88,7 @@ export default function Trigger(props) {
     refLabel,
     size,
   } = state;
-  const triggerButtonId = React.useRef(nanoid());
+  const triggerButtonId = React.useState(nanoid);
```

Comment:
const [triggerButtonId] = React.useState(nanoid);

Reply:
yeap, I think we can replace all nanoid-related tasks in another PR.